builtin/game/falling.lua

```diff
@@ -1,5 +1,7 @@
 -- Minetest: builtin/item.lua
 
+local builtin_shared = ...
```

Comment:
the alternative would be to delete a global variable or member of a global table